test/helpers/alchemist_widgets_binding.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+class AlchemistWidgetsBinding extends AutomatedTestWidgetsFlutterBinding {
+  AlchemistWidgetsBinding({
+    ImageCache? imageCache,
+  }) : _imageCache = imageCache ?? ImageCache();
+
+  final ImageCache _imageCache;
+
+  static WidgetsBinding ensureInitialized() => AlchemistWidgetsBinding();
```

Comment:
Now that I'm thinking about it, maybe we could pass in an instance of `ImageCache` here to `ensureInitialized` and call it from `pumps_test.dart`, just so that it's painfully clear we're initializing our own binding override.